beacon-chain/p2p/discovery.go

```diff
@@ -143,8 +144,10 @@ func (s *Service) createListener(
 		}
 	}
 	dv5Cfg := discover.Config{
-		PrivateKey: privKey,
+		PrivateKey:   privKey,
+		ValidSchemes: enode.ValidSchemes,
```

Comment:
Not super familiar with p2p code. How does `ValidSchemes` relate to this PR?

Reply:
Mostly for readability, we technically do use `enode.ValidSchemes` if it is not set in the config but I figured it would be nicer to explicitly state it here.